standards/architecture-standards.md

```diff
@@ -18,6 +22,10 @@ Various technical standards exist (or are in development) and are relevant to ar
 - [Information security standards](https://educationgovuk.sharepoint.com/sites/how-do-i/SitePages/security-assurance.aspx)
 - Guidance on [meeting accessibility standards](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps#meeting-accessibility-requirements) and the [technical standard for digital accessibility](https://www.gov.uk/guidance/public-sector-website-and-mobile-application-accessibility-monitoring#technical-standard-for-digital-accessibility)
 
+## Standard owners and their responsibilities
+
+## Application of standards
+
 The application of standards is generally assured through co-design in teams, and peer review in architecture and technical communities. There are likely to be additional checks that standards have been met during service assessment. Occasionally, there may be spot checks to ensure consistent and appropriate application of standards.
 
 Architects are responsible for architecture and design decisions in their respective portfolio, service or domain. They should be confident in representing these in their respective communities and any DfE governance or assurance forums. Typically, design decisions are recorded as [Architecture Decision Records (ADRs)](../architecture-documentation/#architecture-decision-records) and stored alongside other design and technical documentation (ideally in Github).
```

Comment:
Yeah, let's leave this in for now, see how the governance pages turn out. Might remove later